### PR TITLE
update for deep learning tools

### PR DESCRIPTION
**Pull request by abhiataero**
addes support for deep learning gptools

### CODE REVIEW

File: scripts/master.xml [hunk 1]
```diff
@@ -1,888 +1,952 @@
...
+						<class_value_field>The name of the field that will contain the class or category label in the output feature class.</class_value_field>
+						<model_arguments>The function arguments defined in the Python raster function class</model_arguments>
+						<max_overlap_ratio>The maximum overlap ratio for two overlapping features. This is defined as the ratio of intersection area over union area. The default is 0.</max_overlap_ratio>
+				</NonMaximumSuppression>
+				<!-- CF Compute Fiducials (Data Management)  -->
+				<!-- https://pro.arcgis.com/en/pro-app/latest/tool-reference/data-management/compute-fiducials.htm -->
+				<ComputeFiducials>
+					<out_fiducial_table>#</out_fiducial_table>
+					<where_clause>#</where_clause>
+                    <fiducial_templates>#</fiducial_templates>#
```
Comment (vijaygit02) [inline scripts/master.xml]: Delete the additional hash

Comment (Chamlika) [reply scripts/master.xml]: @vijaygit02 Yes, although it does not make the XML invalid, we should remove that, thanks!

**Review by Chamlika — APPROVED** [review]
Thanks!

**Comment by abhiataero** [comment]
@vijaygit02  made the changes

**Review by vijaygit02 — COMMENTED** [review]
These tools use IA extension. Does it need to be mentioned anywhere?

**Review by vijaygit02 — APPROVED** [review]

**Comment by vijaygit02** [comment]
@abhiataero  , as it is I am going to update the MDCS doc on github. Shall I add these tools also to it?

**Pull request merged**